util/git-tag-check/git-tag-check.py

```diff
@@ -0,0 +1,117 @@
+import os
```

Comment:
```suggestion
#!/usr/bin/env pythone3

import os
```

Then you can drop the explicit invocation of python directly in the GitHub action workflow, and make it easier to run locally for testing.